packages/webapp/src/utils/credits-purchase-constants.js

```diff
@@ -0,0 +1,12 @@
+const creditPurchaseRoutes = {
+  ESTIMATOR_CREDITS_INDIVIDUAL_ORG: '/credits-purchase/credits-individual-or-organisation'
+}
+
+const views = Object.fromEntries(
+  Object.entries(creditPurchaseRoutes).map(([k, v]) => [k, v.substring(1)])
+)
+
+export default {
+  creditPurchaseRoutes,
```

Comment:
Please replace this with routes following the split into distinct files for credits estimation and credits purchase.